tests/sql/test_temporal_exprs.py

```diff
@@ -88,6 +91,7 @@ def test_extract():
         extract(date from datetimes) as date,
         extract(day from datetimes) as day,
         extract(dayofweek from datetimes) as day_of_week,
+        extract(dayofyear from datetimes) as day_of_year,
```

Comment:
Since you've update to `day_of_week` then we need to update the datetime parts as well.